website/src/types/index.ts

```diff
@@ -15,3 +15,70 @@ limitations under the License.
 */
 
 export type Theme = "default" | "dark";
+
+export type statusDataTypes = {
+    uuid: string;
+    git_ref: string;
+    source: string;
+    started_at: string;
+    finished_at: string;
+    type_of: string;
+    pull_nb?: number;
+    golang_version: string;
+    status: string;
+}
+
+export type prDataTypes = {
+    ID: number;
+    Author: string;
+    Title: string;
+    CreatedAt: string;
+    Base: string;
+    Head: string;
+    error?: any
+}
+
+export interface Range {
+    infinite: boolean;
+    unknown: boolean;
+    value: number;
+}
+  
+export interface MacroDataValue {
+  center: number;
+  confidence: number;
+  range: Range;
+}
+
+export interface ComponentsCpuTime {
+  vtgate: MacroDataValue;
+  vttablet: MacroDataValue;
+}
+
+export interface ComponentsMemStatsAllocBytes {
+  vtgate: MacroDataValue;
+  vttablet: MacroDataValue;
+}
```

Comment:
It feels like these two types could be merged together into, for instance: `ComponentsStats`.